packages/expo-cli/src/commands/build/ios/utils/image.ts

```diff
@@ -1,18 +1,30 @@
+import { Readable } from 'stream';
+
 import fs from 'fs-extra';
 import { PNG } from 'pngjs';
 import pick from 'lodash/pick';
 import { XDLError } from '@expo/xdl';
-import request from 'request';
+import axios from 'axios';
 import validator from 'validator';
 
-export async function ensurePNGIsNotTransparent(imagePathOrURL: string): Promise<void> {
-  let hasAlreadyResolved = false;
-  const stream = validator.isURL(imagePathOrURL, {
+async function getImageStreamAsync(imagePathOrURL: string) {
```

Comment:
We should try to combine this with [image-utils](https://github.com/expo/expo-cli/blob/89ad047fb096c6a17ada65380c7c0fed822135ca/packages/image-utils/src/Download.ts)